src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayToStyles = sourceString
+    .split(';')
+    .reduce((stylesList, stylesParameters) => {
+      const [property, value] = stylesParameters.split(':');
+
+      if (value !== undefined) {
+        stylesList[property.trim()] = value.trim();
+      }
```

Comment:
```suggestion
      if (value) {
        stylesList[property.trim()] = value.trim();
      }
```
You don't need to compare it with undefined, just check if a property exists.